### PR TITLE
Seeder/leecher styling

### PR DESCRIPTION
**Pull request by benwa**
Uses Bootstrap's contextual classes to color the seeder/leecher data instead of inline styles.

Before
screen shot 2017-06-17 at 09 52 30

After
screen shot 2017-06-17 at 09 56 18

### CODE REVIEW

File: nyaa/templates/view.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h3 class="panel-title">
 			</div>
 
 			<div class="col-md-1">Leechers:</div>
-			<div class="col-md-5"><span style="color: red;">{% if config.ENABLE_SHOW_STATS %}{{ torrent.stats.leech_count }}{% else %}Coming soon{% endif %}</span></div>
+			<div class="col-md-5"><span class="text-success">{% if config.ENABLE_SHOW_STATS %}{{ torrent.stats.leech_count }}{% else %}Coming soon{% endif %}</span></div>
```
Comment (maddin77) [inline nyaa/templates/view.html]: Shouldn't this be `text-danger`?

Comment (benwa) [inline nyaa/templates/view.html]: Oops, you're right

**Comment by sharkykh** [comment]
My opinion:
I don't like the colors, they're too dimmed.
And also, these don't work on the dark theme.

**Comment by Arylide** [comment]
Same

**Pull request closed without merging**